netdata/netdata
# Add a CRASH event when the agent fails to properly shutdown

##### Summary
- Store a "START" entry in the database (agent_log)
- Delete the entry during shutdown
- If the entry is detected on startup, the agent is assumed to have terminated abnormally e.g. crashed, killed etc

##### Component Name
code

##### Test Plan
- Start an agent
- Use `kill -9` to kill the agent process
- Restart the agent
  - CRASH event should appear in the posthog analytics
